source/templates.tex

```diff
@@ -7237,13 +7238,13 @@
 \end{note}
 
 \pnum
-The resulting substituted and adjusted function type is used as
-the type of the function template for template argument
-deduction.  If a template argument has not been deduced and its
-corresponding template parameter has a default argument, the
-template argument is determined by substituting the template
-arguments determined for preceding template parameters into the
-default argument. If the substitution results in an invalid type,
+The function type after substitution of explicitly specified template
+arguments with their corresponding template parameters and adjustment
+is used for template argument deduction. If a template argument has
+not been deduced and its corresponding template parameter has a
+default argument, the template argument is determined by substituting
+the template arguments determined for preceding template parameters
+into the default argument. If the substitution results in an invalid type,
```

Comment:
We're trying to use semantic line breaks in any text newly entering the standard. https://sembr.org/
That means (at the very least) that every sentence starts on a new line. Please adjust accordingly.

Reply:
Yeah this paragraph's diff would have been a lot easier to review if there was a line break after the first sentence, because the rest of the paragraph is unchanged. Your rejustification is unhelpful, it makes it look like the entire paragraph has changed.